app/components/smart-banner-test.js

```diff
@@ -0,0 +1,190 @@
+import Ember from 'ember';
+import Thumbnailer from '../modules/thumbnailer';
+import {track, trackActions} from '../utils/track';
+import {system, standalone} from '../utils/browser';
+
+const {
+	$,
+	Component,
+	computed,
+	get,
+	inject,
+	String: {htmlSafe},
+	run,
+} = Ember;
+
+/**
+ * Component for a custom Smart Banner
+ * it's visible only for Android devices
+ * iOS has its own native smart banner - no need to render it there
+ */
+export default Component.extend({
+	classNames: ['smart-banner-test'],
+	classNameBindings: ['noIcon'],
+
+	wikiVariables: inject.service(),
+	currentUser: inject.service(),
+
+	options: {
+		// Language code for App Store
+		appStoreLanguage: 'us',
+
+		// Duration to hide the banner after close button is clicked (0 = always show banner)
+		daysHiddenAfterClose: 15,
+
+		// Duration to hide the banner after it is clicked (0 = always show banner)
+		daysHiddenAfterView: 30,
+	},
+	day: 86400000,
+
+	appId: computed.oneWay(`config.appId.android`),
+	appScheme: computed.oneWay(`config.appScheme.android`),
+	config: computed('wikiVariables', function () {
+		return this.get('wikiVariables').get('smartBanner') || {};
+	}),
+	dbName: computed.reads('wikiVariables.dbName'),
+	description: computed.oneWay('config.description'),
+	icon: computed.oneWay('config.icon'),
+	iconSize: 92,
+
+	iconStyle: computed('icon', function () {
+		if (this.get('noIcon')) {
+			return null;
+		}
+
+		const icon = Thumbnailer.getThumbURL(this.get('icon'), {
+			mode: Thumbnailer.mode.thumbnailDown,
+			width: this.iconSize,
```

Comment:
since there is a `this.get(<whatever>)` everywhere, shouldn't this be retrieved in a similar way?